kymatio/backend/fake_backend.py

```diff
@@ -0,0 +1,5 @@
+from collections import namedtuple
```

Comment:
I thought @janden wanted this removed.

Reply:
So, without this, some tests are not possible to perform. In particulr, one would like to test the setting for which the backend has been created but does not match with the frontend, this being detected thanks to `backend.name`

Reply:
I'm OK with this suggestion

Reply:
Ok so we'll move it to the corresponding test module.